lib/js/atoms/Tile/Tile.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+	<ripple-wrapper :disable="!clickable">
+		<div :class="{ '-clickable': clickable }" class="a-tile">
+			<div v-if="left" class="a-tile__left">
+				<wnl-icon :icon="left" :size="ICON_SIZES.SMALL" />
+			</div>
+			<div class="a-tile__center">
+				<div class="a-tile__eyebrow">
+					<slot name="eyebrow" />
+				</div>
+				<div class="a-tile__title">
+					<slot />
+				</div>
+			</div>
+			<div v-if="clickable" class="a-tile__clickableIcon">
+				<wnl-icon :icon="ICONS.FA_CHEVRON_RIGHT" :size="ICON_SIZES.SMALL" />
+			</div>
+			<div v-else-if="right" class="a-tile__right">
+				<wnl-icon :icon="right" :size="ICON_SIZES.SMALL" />
+			</div>
+			<div v-else-if="text" class="a-tile__text">{{ text }}</div>
+		</div>
+	</ripple-wrapper>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/typography';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/colors/tokens';
+
+::v-deep .ripple {
+	background-color: $color-neutral-ripple !important;
+}
+
+.a-tile {
+	width: 100%;
+	min-height: $space-xl;
+	background-color: $color-neutral-background;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	padding: $space-xxs $space-xs;
+
+	&.-clickable {
+		cursor: pointer;
+
+		&:hover {
+			background-color: $color-neutral-background-hovered;
+		}
+	}
+
+	&__left {
+		margin-right: $space-xs;
+	}
+
+	&__center {
+		display: flex;
+		flex-direction: column;
+		flex-grow: 1;
+	}
+
+	&__clickableIcon {
+		margin-left: $space-xs;
+		color: $color-primary-icon;
+	}
+
+	&__right {
+		margin-left: $space-xs;
+		color: $color-neutral-icon;
+	}
+
+	&__text {
+		@include textXS;
+
+		margin-left: $space-xs;
+		flex-grow: 1;
+		max-width: 30%;
+		color: $color-neutral-text;
+		text-align: right;
+	}
+
+	&__eyebrow {
+		@include textS;
+
+		color: $color-neutral-text-weak;
+	}
+
+	&__title {
+		@include textM;
+
+		color: $color-neutral-text-heavy;
+	}
+}
+</style>
+
+<script lang="ts">
+import RippleWrapper from '../../utils/RippleWrapper.vue';
+import WnlIcon, { ICON_SIZES, ICONS } from '../Icon';
+import { VueConstructor } from 'vue';
+
+export default {
+	name: 'Tile',
+	components: {
+		WnlIcon,
+		RippleWrapper,
+	},
+	props: {
+		clickable: {
```

Comment:
Props doesn't match what we agreed on in https://miro.com/app/board/uXjVOaIWGa0=/?moveToWidget=3458764516053146935&cot=14